Asgardia.Constitution.Proposal.txt

```diff
@@ -1,9 +1,9 @@
 ﻿CONSTITUTION
-OF THE SPACE STATE OF ASGARDIA
+OF THE SPACE NATION OF ASGARDIA

```

Comment:
The use of Nation is much more in line with nations around the world today and has a more clear definition than state. I also used the phrase "Nation State" throughout the document, as this is the common parlance of the United Nations